go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -302,9 +302,19 @@ func (m *ModelClient) create(opModel *OperationModel) ([]ovsdb.Operation, error)
 		})
 	} else if info, err := m.client.Cache().DatabaseModel().NewModelInfo(opModel.Model); err == nil {
 		if name, err := info.FieldByColumn("name"); err == nil {
-			if len(fmt.Sprint(name)) > 0 {
+			var objName string
+			var ok bool
+			if objName, ok = name.(string); !ok {
```

Comment:
@girishmg just a nit, but you could save a line by:

```
objName, ok := name.(string)
if !ok {
```

Reply:
@dcbw done.